From: SJ Park <sj@kernel.org>
Date: Sat, 1 Aug 2026 15:21:48 -0700
Subject: [PATCH] ==== damos_quota_score() division by zero fixes ====

mm/damon: avoid division by zero from damos_quota_score()

DAMON_SAMPLE_MTIER and DAMON_LRU_SORT allow the user to trigger division
by zero in damos_quota_score().  Avoid it by adding parameters
validation checks.

Changes from v1
- v1: https://lore.kernel.org/20260803134034.15217-1-sj@kernel.org
- None.
Changes from RFC v2
- RFC v2: https://lore.kernel.org/20260802162050.89477-1-sj@kernel.org
- Drop RFC tag.
Changes from RFC
- RFC: https://lore.kernel.org/20260801211315.2456-1-sj@kernel.org
- Add a fix for DAMON_LRU_SORT.

Signed-off-by: SJ Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_faedyvro | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_faedyvro

diff --git a/hkml_cv_bogus/hkml_cv_bogus_faedyvro b/hkml_cv_bogus/hkml_cv_bogus_faedyvro
new file mode 100644
index 0000000000000..e69de29bb2d1d
-- 
2.47.3

